### PR TITLE
Patch update

### PR DESCRIPTION
**Pull request by hellozee**

### CODE REVIEW

File: tetrateci/test_1.9.sh [hunk 1]
```diff
@@ -20,4 +20,16 @@ if [[ ${CLUSTER} == "eks" ]]; then
...
+
+for package in $PACKAGES; do
+  n=0
+  until [ "$n" -ge 3 ]
+  do
+    n=$((n+1))
+    sleep 15
+    echo "========================================================TRY $n========================================================"
+    go test -count=1 -p 1 -test.v -tags=integ $package -timeout 30m --istio.test.select=-postsubmit,-flaky ${CLUSTERFLAGS} && break || echo "Test Failed: $package"
+    sudo rm -rf $(ls /tmp | grep istio)
```
Comment (deva26) [inline tetrateci/test_1.9.sh:33]: Here, need to check if n>=3, if so, exit 1 (non zero) to indicate test run failed.

Comment (hellozee) [reply tetrateci/test_1.9.sh:33]: yes but then it would also break the for loop right? is that what we expect?

File: tetrateci/test_1.9.sh [hunk 2]
```diff
@@ -2,11 +2,13 @@
 set -e
 
 git apply tetrateci/patches/common/disable-dashboard.1.9.patch
-git apply tetrateci/patches/common/disable-multicluster.1.9.patch
 git apply tetrateci/patches/common/disable-ratelimiting.1.9.patch
-git apply tetrateci/patches/common/disable-vmospost.1.9.patch
 git apply tetrateci/patches/common/disable-stackdriver.1.9.patch
 
+if $(grep -q "1.17" <<< ${VERSION} ); then
+  git apply tetrateci/patches/common/disable-endpointslice.1.8.patch
```
Comment (deva26) [inline tetrateci/test_1.9.sh:9]: Is this patch same across 1.8 and 1.9? If so, we could rename the file to something generic and add a comment on why we do this. WDYT?

Comment (hellozee) [reply tetrateci/test_1.9.sh:9]: ahh I forgot to port back this commit from my testing branch to change the name, you can actually see that I have a copy of the same patch included in the pr

File: tetrateci/test_1.9.sh (hunk 1 above)
Comment (deva26) [reply tetrateci/test_1.9.sh:33]: Yes, we would fail the test run.

File: tetrateci/test_1.8.sh [hunk 3]
```diff
@@ -29,8 +29,19 @@ if [[ ${CLUSTER} == "aks" ]]; then
...
+  until [ "$n" -ge 3 ]
+  do
+    n=$((n+1))
+    sleep 15
+    echo "========================================================TRY $n========================================================"
+    go test -count=1 -p 1 -test.v -tags=integ $package -timeout 30m --istio.test.select=-postsubmit,-flaky ${CLUSTERFLAGS} && break || echo "Test Failed: $package"
+    sudo rm -rf $(ls /tmp | grep istio)
+  done
+
+  [ "$n" -ge 3 ] && exit 1
```
Comment (deva26) [inline tetrateci/test_1.8.sh:45]: Should be inside the until loop. If it's where we wouldn't know whether the 3rd run was successful or not.

**Review by deva26 — APPROVED** [review]

**Pull request merged**